src/ngraph/runtime/cpu/builder/quantized_conv.cpp

```diff
@@ -193,6 +193,8 @@ namespace ngraph
                     auto& arg0_tensor = external_function->get_tensor_data(args[0].get_name());
                     auto& arg1_tensor = external_function->get_tensor_data(args[1].get_name());
                     auto& arg2_tensor = external_function->get_tensor_data(args[2].get_name());
+                    auto& arg3_tensor = external_function->get_tensor_data(args[3].get_name());
+                    size_t arg3_size = args[3].get_size();
                     auto& arg4_tensor = external_function->get_tensor_data(args[4].get_name());
                     auto& arg5_tensor = external_function->get_tensor_data(args[5].get_name());
```

Comment:
Perhaps out of scope for this PR, but would the code be clearer if these `arg?_tensor` variables were named according to the actual role of each tensor?

Reply:
@cconvey thanks for review/feedback, this PR is WIP needs cleanup, looks like @jbobba wants this to split up also.